Woof/Woof/Helper/DataTrasnformer/DataTransformer.swift

```diff
@@ -29,15 +29,25 @@ enum DataTransformer {
             .capitalized
     }
 
-    /// Converts a numeric value to an integer within the range of 0 to 5, following the zero-to-five style.
+    /// Converts a numeric value to a rating corresponding to a 5-star rating.
     ///
     /// - Parameter value: The numeric value to be converted.
-    /// - Returns: An integer value within the range of 0 to 5.
-    /// If the value is less than 0, 0 is returned.
-    /// If the value is greater than 5, 5 is returned.
-    /// Otherwise, the rounded value is returned. The rounding rule is schoolbook rounding.
-    static func normalizeToZeroToFiveRange(_ value: Double) -> Int {
+    /// - Returns: The rating from 1 to 5 stars from a 5-star rating, or information that the rating is unavailable.
+    /// If the value is less than or equal 0, returned `.unavailable` to indicate an unavailable rating.
```

Comment:
Could you please explain the core logic a bit (why it works in this way)?

Why we return the `unavailable` case for the rating `-5.5`, but return the value `fiveStars` for the rating `10.5`? Should we return `unavailable` if the given value > 5.0?

Reply:
I didn't change core logic and I think it's not a part if this PR.
Created an [issue](https://github.com/ios-course/ironfoudation-team-project/issues/150) and will discuss this with the team.